### ISSUE TITLE
[enhancement] delete application and question

### ISSUE THREAD

**Issue by srobi** [cryocaustik/seat-hr#10]
Enable to delete applications and questions (used only in deleted applications) to clean up and maintain clear profiles

**Comment by kuch3n** [comment]
if one tries to delete an question server anwsers with a 500:

```
[previous exception] [object] (PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`seat`.`seat_hr_answers`, CONSTRAINT `seat_hr_answers_question_id_foreign` FOREIGN KEY (`question_id`) REFERENCES `seat_hr_questions` (`id`)) at /var/www/seat/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:117)
[stacktrace]
```

**Comment by cryocaustik** [comment]
@kuch3n @srobi issue should now be resolved in v1.0.4 with an added feature to allow full application deletion

**Comment by cryocaustik** [comment]
confirmed resolved in discord
